x-pack/plugins/data_visualizer/server/types/deps.ts

```diff
@@ -5,9 +5,8 @@
  * 2.0.
  */
 
-import { Plugin } from 'src/core/server';
+import { SecurityPluginStart } from '../../../security/server';
 
-export class FileDataVisualizerPlugin implements Plugin {
-  setup() {}
-  start() {}
+export interface StartDeps {
```

Comment:
`StartDeps` should live next to `SetupDeps`
This file probably isn't needed and `StartDeps` can be moved to the `plugin.ts`

Reply:
I think originally StartDeps was in a different `types` in order avoid circular dependencies because it was also used in `server/routes`. For now, I moved `SetupDeps` into the same file for better consistency.